patterns/1-initial/capacity-for-contributions.md

```diff
@@ -0,0 +1,69 @@
+## Title
+
+Managing capacity for reviewing contributions
+
+## Patlet
+
+Reviewing InnerSource contributions takes time and effort. This should be reflected in capacity planning, especially for larger contributions. Expectations and available capacity should be transparent so that contributors understand when their contributions will be reviewed and released.
```

Comment:
Can a team assume that any contribution will be "released"?
i.e. it could also be rejected entirely, in theory, right?

Reply:
@spier sure, contributions could be rejected for different reasons.
I don't feel strongly either way here, if we should mention it at this point or not.

Reply:
```suggestion
Reviewing InnerSource contributions takes time and effort. This should be reflected in capacity planning, especially for larger contributions. Expectations and available capacity should be transparent so that contributors understand when their contributions will be reviewed.
```

I am suggesting this change, as the host team likely does not promise a release. They only promise a review :)

Reply:
The reason I mentioned 'released' is because this is often what really matters to the contributor. It's all very well getting a speedy review, but if this contribution is to unblock some other work, or fix an error in production, they need to know when they might expect a release. Especially if contributing to software that is on a slow e.g. monthly release cycle.

Suggested rewording:

```suggestion
Reviewing InnerSource contributions takes time and effort. This should be reflected in capacity planning, especially for larger contributions. Expectations and available capacity should be transparent so that contributors understand when their contributions will be reviewed and, if accepted, released.
```

Reply:
👍